src/frontend/src/pages/AdminToolsPage/AdminToolsFinanceConfig.tsx

```diff
@@ -1,11 +1,12 @@
-import { Grid } from '@mui/material';
-import PageBlock from '../../layouts/PageBlock';
+import { Grid, Typography } from '@mui/material';
 import VendorsTable from './FinanceConfig/VendorsTable';
 import AccountCodesTable from './FinanceConfig/AccountCodesTable';
 
 const AdminToolsFinanceConfig: React.FC = () => {
   return (
-    <PageBlock title="Finance Config">
+    <div>
+      <Typography variant="h4">Finance Config</Typography>
+      <br />
       <Grid container spacing="3%">
```

Comment:
this will then become the parent container for everything, add some horizontal padding